packages/client/src/generators.ts

```diff
@@ -0,0 +1,37 @@
+import { fromByteArray } from 'base64-js';
+import { sha256 } from 'js-sha256';
+
+function generateRandomData(length: number) {
+  // Use web crypto APIs if possible
```

Comment:
how about we just use crypto directly like [playground](https://github.com/logto-io/playground/blob/master/src/pages/Home/index.tsx#L28), and do backward compatibility work afterwards? actually the web crypto api is [widely supported](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API#browser_compatibility) (even in IE 11)

Reply:
and no need for `js-sha256` dependency since using subtle is enough